demo/index.html

```diff
@@ -68,8 +68,8 @@ <h1>Vbrick SDK Embed Test</h1>
 					<div>
 						<label for="embedType">Embed Type</label>
 						<div class="radio-grid">
-							<label><input type="radio" name="embedType" value="vod" checked>VOD</label><span></span>
-							<label><input type="radio" name="embedType" value="webcast">Webcast</label><span></span>
+							<label><input type="radio" name="embedType" id="embedType" value="vod" checked>VOD</label>
+							<label><input type="radio" name="embedType" id="embedType" value="webcast">Webcast</label>
```

Comment:
It isn't valid HTML to have two elements with the same id. the `form.elements` convention uses the name value, FWIW

Reply:
ok how about document.forms["demoform"].elements['embedType'].value == 'vod';

Reply:
your code `form.elements.embedType.value == 'vod'`  is fine already, though that suggestion works as well. I was just noting that you don't need the `id` parameters.

Reply:
Thanks, I'm all for setting a good example with our sdk demo